tests/unit/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Every source file (and header file) should be listed here.
+# If you add a new file, add it to this list.
```

Comment:
Can't we automate this (by just specifying complete folders)?

Reply:
We can't if we don't want to have to recreate the entire project every time a file is added.